hyperspy/misc/signal_tools.py

```diff
@@ -56,11 +63,83 @@ def are_signals_aligned(*args, ignore_axis=None):
     return True
 
 
+def _check_axes_calibration(ax1, ax2):
+    """Check if the calibration of two axes matches.
```

Comment:
This equality operation could be useful for other operations and a better place for this function may be `hyperspy.axes` or an utils module?
A note for later, this would be to be adapted in the `non_uniform_axes`.

Reply:
I've moved it to `hyperspy/misc/axis_utils.py`. Had a bit of trouble with circular imports so haven't made it any more complicated.